WPFHospitalEditor/SearchResultDialog.xaml.cs

```diff
@@ -37,20 +46,24 @@ private void CreateDynamicGrid()
 
         private void createRowContent()
         {
+            row.Clear();
+            int i = 2;
```

Comment:
What this "i" should to represent?

Reply:
It represents key of dictionary, I changed its name.